controllers/suite_test.go

```diff
@@ -288,3 +288,10 @@ func createTestClusterReconciler() *FoundationDBClusterReconciler {
 		LockClientProvider:  NewMockLockClient,
 	}
 }
+
+func testInstanceLabels(instanceName string, instanceClass fdbtypes.ProcessClass) map[string]string {
```

Comment:
What is the reason that you added this function but only use it in one case?

Reply:
Ah, I was trying to use it in 3 places but then saw that the labels were actually subtly different. I can unwind this function, or look more aggressively for places to use it....

Reply:
I think removing it makes the most sense in the current state, w can revisit this when we change something, or someone wants to refactor the tests :)

Reply:
Fixed, and fixed the semantic conflict that had been merged to master

Reply:
And fixed the conflict with #518 too - this is just large, not complex, would love to see it merged to stop the conflict chasing :)